crates/bevy_core_pipeline/src/bloom/mod.rs

```diff
@@ -159,7 +159,7 @@ impl Default for BloomSettings {
             threshold: 1.0,
             knee: 0.1,
             scale: 1.0,
-            intensity: 1.0,
+            intensity: 0.3,
```

Comment:
Rather than change it to 0.3, I'd rather multiply by 0.3 in the shader and leave this as 1.0. 

If we do keep it as 0.3, please update the documentation comments in BloomSettings accordingly.

Reply:
I saw the PR this morning and thought the opposite - why have a seemingly arbitrary fudge factor in the engine rather than seeing the default value to 1.0/3.0?

Reply:
I'm less certain about this approach. Adjusting parameters so the defaults are always 1.0 doesn't seem particularly helpful. And its not like we're normalizing these scales to be bounded by [0, 1]. We're just arbitrarily rescaling. If we ever decide to change the default value, we shouldn't need to adjust the scale to do that.

Reply:
@mockersf @cart not a big deal but curious what you think.

Reply:
Yeah I would prefer have no fudge factor and instead set the default to the preferred (unscaled) value.

Reply:
(My first comment was sent without refreshing the page / seeing @superdump's first comment. Good to know that we're on the same page here).

Reply:
back to 0.3 it goes!